tests/environment-setup/test_used_python.py

```diff
@@ -39,8 +37,20 @@ def make_versions_comparable(*versions) -> List[Tuple[int]]:
     return [v[:n] for v in vers]
 
 
+@pytest.fixture(scope="session")
+def expected_python_version(osparc_simcore_root_dir: Path) -> Tuple[int, ...]:
```

Comment:
I do not understand the idea behind this test

Reply:
this is a fixture (not a test)
it is responsible to identify the main python version of the repo
it is very convenient to do python upgrades
remember that this is a part of the py3.8 PR